README.md

```diff
@@ -1,57 +1,3 @@
-README
-======
-
-What is Symfony?
------------------
-
-Symfony is a PHP full-stack web framework. It is written with speed and
-flexibility in mind. It allows developers to build better and easy to maintain
-websites with PHP.
-
-Symfony can be used to develop all kind of websites, from your personal blog
-to high traffic ones like Dailymotion or Yahoo! Answers.
-
-Installation
-------------
-
-The best way to install Symfony is to use the [official Symfony Installer][7].
-It allows you to start a new project based on the version you want.
-
-Documentation
--------------
-
-The "[Quick Tour][1]" tutorial gives you a first feeling of the framework. If,
-like us, you think that Symfony can help speed up your development and take
-the quality of your work to the next level, read the official
-[Symfony documentation][2].
-
-Contributing
-------------
-
-Symfony is an open source, community-driven project. If you'd like to contribute,
-please read the [Contributing Code][3] part of the documentation. If you're submitting
-a pull request, please follow the guidelines in the [Submitting a Patch][4] section
-and use [Pull Request Template][5].
-
-Community Reviews
------------------
-
-If you don't feel ready to contribute code or patches, reviewing issues and pull 
-requests can be a great start to get involved and give back. In fact, people who 
-"triage" issues are the backbone to Symfony's success!  
-More information can be found in the [Community Reviews][8] guide.
-
-Running Symfony Tests
-----------------------
-
-Information on how to run the Symfony test suite can be found in the
-[Running Symfony Tests][6] section.
-
-[1]: https://symfony.com/doc/current/quick_tour/index.html
-[2]: https://symfony.com/doc/current/
-[3]: https://symfony.com/doc/current/contributing/code/index.html
-[4]: https://symfony.com/doc/current/contributing/code/patches.html#check-list
-[5]: https://symfony.com/doc/current/contributing/code/patches.html#make-a-pull-request
-[6]: https://symfony.com/doc/master/contributing/code/tests.html
-[7]: https://symfony.com/doc/current/book/installation.html#installing-the-symfony-installer
-[8]: https://symfony.com/doc/current/contributing/community/reviews.html
+<div align="center">
```

Comment:
Markdown doesn't care with positioning. It has to be done with HTML or the image could be changed to Markdown but then I purpose to use [this image](http://symfony.com/logos/symfony_black_02.svg?v=4) and just leave it at the left.